app/src/main/java/com/junewon/sopthousparty/util/BindingAdapters.kt

```diff
@@ -0,0 +1,11 @@
+package com.junewon.sopthousparty.util
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import coil.load
+
+@BindingAdapter("app:imageUrl")
+fun ImageView.setImageUrl(imageUrl: String?) {
+    if (imageUrl == null || imageUrl == "") return
+    load(imageUrl)
+}
```

Comment:
바인딩어뎁터는 사랑이죠~~ 코일 사용도 아주 좋습니다

Reply:
imageUrl에 null 던져주셔도 됩니다.

<img width="503" alt="스크린샷 2022-09-20 오후 5 41 19" src="https://user-images.githubusercontent.com/54518925/191211048-3f16fbb2-dfb7-4729-9ed4-31c5d559f589.png">
<img width="497" alt="스크린샷 2022-09-20 오후 5 41 36" src="https://user-images.githubusercontent.com/54518925/191211121-3512cebc-4d07-4766-b2fe-0bc1c8cca9e5.png">
<img width="559" alt="스크린샷 2022-09-20 오후 5 41 50" src="https://user-images.githubusercontent.com/54518925/191211183-ea86db08-0db8-4a68-826f-ff4f8cc17363.png">
<img width="400" alt="스크린샷 2022-09-20 오후 5 42 02" src="https://user-images.githubusercontent.com/54518925/191211246-da87d51d-8ae5-4193-8d6e-3177914b9c00.png">

코일에서 내부 구현을 잘 살펴보시면 data가 null이 들어올 시에 ImageRequestBuilder에서 NullRequestData를 넣어주네요

Reply:
[공식문서](https://coil-kt.github.io/coil/getting_started/)에서도 해당 내용에 대해 서술하고 있습니다.

> If you do not set data, it will default to [NullRequestData](https://coil-kt.github.io/coil/api/coil-base/coil.request/-null-request-data).

만약 데이터가 null이라면 fallback에 해당하는 데이터 넣어줄 수 있다고 하니 참고하면 좋을 것 같습니다.

<img width="540" alt="스크린샷 2022-09-20 오후 5 48 47" src="https://user-images.githubusercontent.com/54518925/191212839-586c099f-a3a0-46ca-bb10-596daaefcb1b.png">

Reply:
좋은 정보 알아갑니다~~ 최곱니다

Reply:
와아... 진짜 개꿀정보네요.. coil 공식문서도 한 번 읽어봐야겠어요 !!